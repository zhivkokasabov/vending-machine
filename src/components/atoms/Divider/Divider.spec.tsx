import { render } from "@testing-library/react";
import { Divider } from ".";

describe("Divider tests", () => {
  const renderComponent = () => render(
    <Divider />
  );

  it("should render", () => {
    const { asFragment } = renderComponent();

    expect(asFragment()).toMatchSnapshot();
  });
});
