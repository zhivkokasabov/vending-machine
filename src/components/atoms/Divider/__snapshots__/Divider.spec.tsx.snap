// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Divider tests should render 1`] = `
<DocumentFragment>
  <div
    class="w-full h-[1px] bg-black my-4"
  />
</DocumentFragment>
`;
